lib/daru/io/exporters/base.rb

```diff
@@ -0,0 +1,19 @@
+require 'daru/io/base'
+
+module Daru
+  module IO
+    module Exporters
+      class Base < Base
```

Comment:
It's `Daru::IO::Importers::Base` inheriting from `Daru::IO::Base`. 

`Importers::Base` acts as parent class to all importers, providing importer-specific helper functions.

`Base` acts as parent class to `Importer::Base` and `Exporter::Base`< providing generic functions like `optional_gem`. Should the code be changed be explicitly specify `Base < Daru::IO::Base`?